client/my-sites/post-relative-time-status/index.jsx

```diff
@@ -47,9 +47,9 @@ module.exports = React.createClass( {
 		return (
 			<span className="post-relative-time-status__time">
 				<Gridicon icon="time" size={ 18 } />
-				<span className="post-relative-time-status__time-text">
+                <time className="post-relative-time-status__time-text" dateTime={ time } title={ time }>
```

Comment:
There is whitespace inconsistency here. The Calypso code base uses tabs, not spaces.